AVRF.md

```diff
@@ -181,22 +183,22 @@ This event will occur **_before_** any other events.
 
 ## AVrf custom provider installation
 
-Copy your provider dll to the %systemroot%\system32 (note for WOW64 you should use appreciate folder syswow64 and registry keys). Administrator rights required.
+Copy your provider dll to the %SystemRoot%\System32 directory (note: for x86 you have to use the SysWOW64 (SysWOW64 contains x86 files, System32 contains x64 files. No, it is not the other way around.) directory and registry keys). Administrator rights are required.
```

Comment:
The original text was correct, this applies to WoW64 only. On native x86 Windows, System32 contains x86 files.

Reply:
What does the text "For WOW64" mean anyway? Is it implying that if you want to hook into 32 bit processes on x64 systems, you should use the SysWOW64 diectory and instead of HKLM\Software\..., you should use HKLM\Software\WOW6432Node instead?

Reply:
yes

Reply:
I've updated it.